### DISCUSSION TITLE
Way to escape bracketed numbers in clue

### DISCUSSION THREAD

**Discussion by billykenny** [viresh-ratnakar/exolve#70] [Q&A] [answered]
I tried to enter this clue: 

Writer (50) caught in swindle processing ore (8) 

but exolve sees the (50) as the clue enumeraotr and just prints

Writer (50)

Is there any way to get around this restriction?

**Comment by Antagony1060** [comment]
I would just replace the brackets with [HTML character codes](https://www.rapidtables.com/web/html/html-codes.html): `&#40;50&#41;`

**Reply by billykenny** [reply]
Thanks for that - that works fine, but it means keeping two different versions of clues, one for exolve and one for Acrosslite for example, as the latter would reproduce the HTML code. (I use Acrosslite to create PDF versions of the crosswords)

**Reply by Antagony1060** [reply]
Honestly, if you're only using AcrossLite to create PDF files, you're doing unnecessary work. Most modern browsers have a "Save to PDF" option in their print functions, which handle the job very well, preserves the integrity of any HTML formatting, and allow links within the page. 

For example, I used MS Edge to save this: https://www.dropbox.com/s/s45koj0frn7nuxj/POTD_032.pdf?dl=1

**Reply by billykenny** [reply]
I didn't know about the "Save to PDF" browser option - that should do the trick. Thanks so much for your help!

**Comment by viresh-ratnakar** [comment, marked as answer]
+1 to Antagony1060's remark: Exolve already has code to generate reasonable print-outs and PDFs. If you see anything odd or would like to see something done better in that, I'll be happy to take a look.

In any case, I have also pushed Exolve v1.20 which now uses the *last* enum-like part as the enum, if there are several choices. Corner cases are dealt with as follows (copying from the CHANGELOG), for completeness:

- If a clue has multiple enum-like parts, use the *last* one.
- However, override that if there is an earlier enum-like substring
  that is immediately (or with intervening spaces) followed by "[...]"
  (which is a tell-tale start of the annotation part).
- Allow an empty "[]" to be placed to mark the end of the clue part
  for dealing with ambiguities and also for the corner case that
  the anno has to start with "[...]" without making the contents of
  the square brackets be treated like the solution. This used to
  require explicitly providing the solution (like
  "... clue (6) [WITTER] [t]WITTER") but can now also be tackled
  with "... clue (6) [] [t]WITTER". The empty "[]" is not shown in the
  clue or the anno.

**Reply by billykenny** [reply]
Great that V1.20 handles the enum problem - many thanks for that.

Re PDFs, if I right click and select Print to PDF (I'm using Windows 10 Pro and MS Edge, and not changing any settings) from the Exolve puzzle, the resulting printout occupies just about half the page, the font is quite small, as is the grid.

Acrosslite produces a larger, more user friendly output (in terms of writing in answers, although it doesn't look as elegant).

These links are to two PDF versions of the same puzzle:

[Acrosslite version](https://www.hubblecrosswords.com/aclite033.pdf)

[Exolve version](https://www.hubblecrosswords.com/ex033.pdf)
